man/systemd.service.xml

```diff
@@ -897,6 +897,28 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>RestartMode=</varname></term>
+
+        <listitem>
+          <para>Takes a string value that specifies how a service should restart:
+            <itemizedlist>
+              <listitem><para>If set to <option>normal</option> (the default), the service restarts by
+              going through a failed/inactive state.</para></listitem>
+
+              <listitem><para>If set to <option>quick</option>, the service restarts without
+              going through a failed/inactive state.
```

Comment:
"If set to quick, the service transitions to the activating state directly during auto-restart, skipping failed/inactive state."

Maybe something like this is clearer?